components/card/card.css

```diff
@@ -0,0 +1,244 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+.main-container {
+  margin: 2rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  gap: 1rem;
+  align-items: center;
+}
+
+.card {
+  width: 15rem;
+  border-radius:var(--border-md);
+  cursor: pointer;
+  box-shadow: rgba(90, 90, 90, 0.35) 0px 5px 15px;
+  position: relative;
+}
+
+.card-badge {
+  background-color: var(--darkOrange);
+  color: var(--white);
+  position: absolute;
+  display: inline-block;
+  padding: 0.4rem 0.5rem;
+  font-weight: var(--font-semiBold);
+  font-size: 0.8rem;
+  left: 0;
+  top:2%;
+}
+
+.card:hover {
+  box-shadow: rgba(19, 18, 18, 0.35) 5px 15px 15px;
+  transition: 0.3s;
+}
+
+.card-img {
+  border-radius: var(--border-md) var(--border-md) 0 0;
+  max-width: 100%;
+  object-fit: cover;
+}
+
+.card-info {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+}
+
+.card-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card-title h3 {
```

Comment:
Avoid grabbing elements in a component library, try using classes in place of it. Doing so can affect the user's styling.